src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Represents the user interface of the Duke program.
+ */
+public class Ui {
+    private static String BOT_NAME = "DIGITAL DADDY";
+    private static String EMOJI = "\uD83E\uDD16";
+    private static String LINE_SEPARATOR =
+            "____________________________________________________________";
+    private Scanner sc;
+
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    public String readCommand() {
```

Comment:
Should this method be renamed to better reflect the return value?
```suggestion
    public String getCommand() {
```